build/media-suite_compile.sh

```diff
@@ -2,7 +2,7 @@
 # shellcheck disable=SC2034,SC1090,SC1117,SC1091,SC2119
 shopt -s extglob
 
-FFMPEG_BASE_OPTS=(--pkg-config-flags=--static)
+FFMPEG_BASE_OPTS=("--pkg-config-flags=--static")
```

Comment:
What's the issue with these?

Reply:
Shellcheck keeps complaining about the = being literal due to thinking we are trying to use associative arrays